src/lib/eap_aka_sim/encode.c

```diff
@@ -992,56 +935,47 @@ ssize_t fr_aka_sim_encode(REQUEST *request, VALUE_PAIR *to_encode, void *encode_
 	}
 	fr_cursor_head(&cursor);	/* Reset */
 
-	MEM(p = buff = talloc_zero_array(eap_packet, uint8_t, 1024));	/* We'll shrink this later */
-	end = p + talloc_array_length(p);
+	MEM(buff = talloc_zero_array(eap_packet, uint8_t, 1024));	/* We'll shrink this later */
+	fr_dbuff_init(&dbuff, buff, talloc_array_length(buff));
```

Comment:
should now be an extensible dbuff, and should probably start at 512

Reply:
Done.